js/app/widgets/lightbox.js

```diff
@@ -275,14 +274,14 @@ const LightboxContainer = new Knowledge.Class({
         this._previous_button = new Gtk.Button({
             halign: Gtk.Align.START,
             valign: Gtk.Align.START,
-            margin: this._ICON_MARGIN,
+            margin: 20,
             image: img_prev
         });
         Utils.set_hand_cursor_on_widget(this._previous_button);
         this._previous_button.connect('clicked', function () {
             this.emit('navigation-previous-clicked');
         }.bind(this));
-        this._previous_button.get_style_context().add_class(nav_button_class);
+        this._previous_button.get_style_context().add_class(Utils.get_element_style_class(Lightbox, 'navigationButtonPrevious'));
```

Comment:
@ptomato what's the 'same comment' you're talking about? 
and good thought. good thought. changed it to `navigationPrevious` and `navigationNext`.